code/modules/storage/items/firstaid.dm

```diff
@@ -0,0 +1,7 @@
+// allows first aid kits to hold pill bottles
+/datum/storage/firstaid
+
+/datum/storage/firstaid/check_can_hold(obj/item/W)
+	. = ..()
+	if(istype(W, /obj/item/storage/pill_bottle))
+		return src.get_fullness(W)
```

Comment:
Is there a reason to override the proc rather than just using the `can_hold` list?

Reply:
☝️

Reply:
misread the order of operations for can_hold ^^'